web/packages/dmt-core/src/components/Pickers/EntityPickerButton.tsx

```diff
@@ -23,6 +23,7 @@ export const EntityPickerButton = (props: {
   scope?: string // Path to a folder to limit the view within
 }) => {
   const { onChange, typeFilter, text, variant, scope } = props
+  // @ts-ignore
```

Comment:
the type error should be fixed in master branch, and not necessary to add the @ts-ignore